setup.py

```diff
@@ -38,6 +38,7 @@
     install_requires=[
         'audioread >= 2.0.0',
         'numpy >= 1.15.0',
+        'packaging >= 20.3',
```

Comment:
Thanks -- Is 20.3 the minimal version required here?  From a skim of the docs, it looks like anything above 17 would do the trick, but I don't have much experience with this package.

Reply:
That's what I had in my virtualenv, totally not required I think. I can put 17.